buildSrc/src/main/kotlin/io/getstream/chat/android/Configuration.kt

```diff
@@ -1,8 +1,8 @@
 package io.getstream.chat.android
 
 object Configuration {
-    const val compileSdk = 31
-    const val targetSdk = 31
+    const val compileSdk = 32
+    const val targetSdk = 32
```

Comment:
We shouldn't bump compile and target SDK versions for now as it's a breaking change and we are not ready to do that as we didn't adapt to new permissions system, etc.

Reply:
I was concern about that as well, but reviewing documentation [SDK 32 is Android 12L](https://developer.android.com/studio/releases/platforms#12), the new permissions system is added on [SDK 33 (Android 13)](https://developer.android.com/about/versions/13/behavior-changes-13)

Reply:
@JcMinarro nice catch! We should be good to bump both then